nearuplib/nodelib.py

```diff
@@ -401,25 +420,26 @@ def setup_and_run(nodocker, binary_path, image, home_dir, init_flags, boot_nodes
         run_nodocker(home_dir, binary_path, boot_nodes, verbose, chain_id)
     else:
         run_docker(image, home_dir, boot_nodes, verbose)
+        print("Node is running! \nTo check logs call: docker logs --follow nearcore")
 
 
 def stop():
     if shutil.which('docker') is not None:
         out = subprocess.check_output(
             ['docker', 'ps', '-q', '-f', 'name=nearcore'], universal_newlines=True).strip()
-        if out != '':
-            stop_docker()
+        if out:
+            stop_docker(out.split('\n'))
         else:
             stop_native()
     else:
         stop_native()
 
 
-def stop_docker():
-    """Stops docker for Nearcore and watchtower if they are running."""
-    docker_stop_if_exists('watchtower')
+def stop_docker(containers):
+    """Stops docker for Nearcore if they are running."""
     print('Stopping docker near')
-    docker_stop_if_exists('nearcore')
+    for c in containers:
```

Comment:
There is no real need to split the container names using `\n` . `docker stop` accepts the list of containers. I frequently use `docker stop $(docker ps -q)`

Reply:
i changed to ['docker', 'stop', *names] now. interestingly docker ps -q inside $() become a space separated list instead of \n separated. But in python i can only get \n separated result